validmind/errors.py

```diff
@@ -339,7 +339,7 @@ def raise_api_error(error_string):
     try:
         json_response = json.loads(error_string)
         api_code = json_response.get("code")
-        api_description = json_response.get("description")
+        api_description = json_response.get("description", json_response.get("message"))
```

Comment:
This is the reason all those API errors were so cryptic, we were not accessing the correct element.

Reply:
oops 🙈